modules/apigee_edge_apidocs/apigee_edge_apidocs.links.menu.yml

```diff
@@ -0,0 +1,13 @@
+# API Doc menu items definition
+entity.apidoc.collection:
+  title: 'API Docs'
+  route_name: entity.apidoc.collection
+  description: 'List API Docs'
+  parent: system.admin_structure
+  weight: 100
+
+apidoc.admin.structure.settings:
+  title: 'API Docs'
+  description: 'Configure API Doc settings'
+  route_name: apidoc.settings
```

Comment:
Are you sure this route name is right? I would have rather expected something like `apigee_edge_apidocs.settings` as name.